ts/store/actions/lollipop.ts

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
For actions/reducers and any other kind of implementations specific to lollipop what about having a folder inside `features` main folder? what do you think?

Reply:
Sure!